python/semantic_kernel/plugin_definition/parameter_view.py

```diff
@@ -11,8 +13,8 @@ class ParameterView(SKBaseModel):
     name: str
     description: str
     default_value: str
-    type_: str = Field(default="string", alias="type")
-    required: bool = False
+    type_: Optional[str] = Field(default="string", alias="type")
```

Comment:
Why this change?

Reply:
As part of the `prompt_template_config` changes I did to support plugins (like those use by dotnet in the root of the SK directory), the `"input_variables"` block was never hit. The `type_` and `required` params appear to be optional in those `"input_variables"`. Updated this here to support that.